docs/SUPPORT.md

```diff
@@ -0,0 +1,60 @@
+# Customer support 
+
+### Install build of SDK from a git branch 
+
+While helping you resolve your issue, we may have some code that we would like for you to test out for us to see if it solves your issue. To install this code into your iOS app, follow these instructions below. 
+
+> Note: We do not recommend that you ship your app to production with a build of our SDK from a branch. Only ship your app to production with [official releases](https://github.com/customerio/customerio-ios/releases) of our SDK. 
+
+For demonstration purposes, let's say that our support team has told you to install the git branch name `demo-bug-fix`. In the instructions below, replace `demo-bug-fix` with the branch name our support team has sent to you. 
+
+* **Did you install our SDK into your app via Swift Package Manager (SPM)?**
+
+Assuming that you have already [followed our instructions for installing our SDK via SPM](https://customer.io/docs/sdk/ios/getting-started/#install), navigate to Xcode > open your project settings > Package Dependencies tab > double click the `customerio-ios` SDK > select the "Branch" option, and enter in the branch name `demo-bug-fix` > Select "Done". 
+
+If you have already followed these instructions and have installed a specific branch of the SDK, make sure that you have the latest changes of this branch installed. From Xcode > open "File" > Packages > Resolve Package Versions. 
+
+* **Did you install our SDK into your app via Cocoapods?**
+
+In your `Podfile`, replace any existing Customer.io entries that you have with these new entries:
+```ruby
+# Replace these:
+pod 'CustomerIO/Tracking', "~> 2.0.0"
+pod 'CustomerIO/MessagingPushFCM', "~> 2.0.0"
+
+# With these:
+branch_name = 'demo-bug-fix'
+pod 'CustomerIOCommon', :git => 'https://github.com/customerio/customerio-ios.git', :branch => branch_name
+pod 'CustomerIOMessagingPush', :git => 'https://github.com/customerio/customerio-ios.git', :branch => branch_name
+pod 'CustomerIOTracking', :git => 'https://github.com/customerio/customerio-ios.git', :branch => branch_name
+pod 'CustomerIOMessagingPushFCM', :git => 'https://github.com/customerio/customerio-ios.git', :branch => branch_name
```

Comment:
I think it is good to mention that replace `CustomerIOMessagingPushFCM` with `CustomerIOMessagingPushAPN` if the customer is using APN for push notifications.